docs/docs/concepts/policies.mdx

```diff
@@ -56,99 +65,73 @@ The _Flow Policy_ is part of Rasa's new
 with version `3.7.0`.
 :::
 
-The Flow Policy is a state machine that allows you to manage and control your
-assistant's conversational flows. It facilitates the integration of your business
-logic with machine learning capabilities.
+The Flow Policy is a state machine that deterministically executes
+the business logic defined in your [flows](./flows.mdx).
 
 The Flow Policy oversees your assistant's state, handles state transitions, and
-triggers new flows when needed. The "Dialogue Stack" represents all started but
-incomplete flows and refers to steps defined in the
-[flows specification](../concepts/flows.mdx).
-
-Using the Flow Policy, you can encapsulate your logic into flows, thereby
-guiding the conversation based on user input and system response. Other
-classifiers adjust the underlying state machine to handle edge cases.
-
-For more detailed information on defining flows, refer to the
-[flows specification](../concepts/flows.mdx).
+triggers new flows when needed.
```

Comment:
this line makes it sound like it would start flows autonomously and regularly. It only does it itself in the case of intents and links. Maybe to not introduce too much detail upfront: [...] handles state transitions, and in a few situations triggers new flows.